app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+
+  def new
+    
+    @user = User.new
+  end
+
+  def create
+    
+    @user = User.new(params.require(:user).permit!)
```

Comment:
Do you need this instance variable? It looks like you could just do it with a local variable.